test/tests/functional/pbs_advance_resv_end_hook.py

```diff
@@ -0,0 +1,169 @@
+from tests.functional import *
+import time
+
+
+class TestAdvanceReservationEndHook(TestFunctional):
+    """
+    Tests to verify the reservation end hook for
+    an advanced reservation if it ends or gets deleted
+    """
+
+    def setUp(self):
+        """ Create and import the python script for reservation end hook
+            in setup method"""
+
+        super(TestAdvanceReservationEndHook, self).setUp()
```

Comment:
All other tests I've seen call TestFunctional.setUp().  I think this is equivalent.  I'd rather see consistency with other tests though.